### ISSUE TITLE
Textures missing / shader error after installing FF 3.5.1

### ISSUE THREAD

**Issue by 0xchristom** [ThirteenAG/GTAIV.EFLC.FusionFix#701]
### What should happen

Hi, after I installed the latest Fusion Fix 3.5.1 everything worked well. Then by a mistake, I've added a few mods (textures, dxvk, radio change, and this https://www.nexusmods.com/gta4/mods/385 [) 

now streets look like this:
![ff gta4](https://github.com/user-attachments/assets/d9d3832c-641a-4423-8adc-3c2cd3074eb6)

I already tried multiple clean installation combinations, and it only comes back to its natural form if I don't use Fusion Fix at all:
![ff gta4 1](https://github.com/user-attachments/assets/59ee62d5-ac29-4ab3-9c69-70d15c91bf76)

List of mods I've used after the clean install and the issue persists:
Fusion Fix 3.5.1
Console Visuals Vegetation

I cleaned appdata folders of the game, no idea what else to do. 

### Checklist

- [X] Check the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already implemented.
- [X] [Search for other requests](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same feature.

**Comment by Kiryusha2Pizza** [comment]
Same here, installed FF last version 3 days ago, everything was perfect. But I didn't even install any other mods. Just started the game today, didn't change any setting but got some kind of dirt all over the roads, trees and cars.
![dirt1](https://github.com/user-attachments/assets/a06c8bc2-9b36-4405-bf86-eb05dcb5d101)
![dirt2](https://github.com/user-attachments/assets/66a5ac4b-65a3-4486-b045-08b7e8fcd772)

**Comment by dymck1** [comment]
Yes same issue here. Interestingly if I change the resolution in the graphics settings, everything goes back to normal for a moment, but once I exit back out of the menu, it glitches out again. Very strange.

**Comment by dymck1** [comment]
Just figured it out! This isn't a glitch but is supposed to be snow! You can turn it off by dialing 766-555-0100 in game. I guess this is a seasonal thing they do. Strange that there isn't an option in the menu to disable it, especially considering it doesn't really look like snow, just looks like more like a glitch to me. Either way when I dialed that number it went back to normal.

**Comment by 0xchristom** [comment]
Can confirm, dialing the number from the "in-game phone" works! Closing. Well done everyone.

**Comment by Ash-735** [comment]
Yes, we adapted GTAO's Snow effect in secret and it was supposed to be a nice surprise but sadly we've got a lot of hate and negative feedback over this so it will be removed in a future update. We're sorry for all of you that decided to start playing GTA IV's storymode during this couple of days causing such confusion.

We've learnt our lesson and we won't be adding anymore features to this game.

**Comment by juicermv** [comment]
> We've learnt our lesson and we won't be adding anymore features to this game.

That's definitely the wrong lesson to learn lmao. 

The actual lesson is don't implement and then force something onto users without properly communicating its existence.

**Comment by Ash-735** [comment]
> > We've learnt our lesson and we won't be adding anymore features to this game.
> 
> That's definitely the wrong lesson to learn lmao.
> 
> The actual lesson is don't implement and then force something onto users without properly communicating its existence.

Kinda ruins the surprise aspect of it, plus this was only on for just over a day, out of the rest of the time of the year, you'd think the snow falling would give it away as to what it is! But alas, no, soo many of you sent a lot of hate over this that it's just deflated any desire to continue work on this right now, so next update the Snow will be gone.

**Comment by juicermv** [comment]
> Kinda ruins the surprise aspect of it, plus this was only on for just over a day, out of the rest of the time of the year, you'd think the snow falling would give it away as to what it is! But alas, no, soo many of you sent a lot of hate over this that it's just deflated any desire to continue work on this right now, so next update the Snow will be gone.

For the record I was aware this was added because I've been following the commits and PRs of this project on a regular basis lol, and there's been no hate from my end.

I get that the intentions of this little feature were pure and all but I think people's issue with it has mostly been the execution. 

It's not mentioned properly anywhere that this is even a thing, or how to turn it off.
Calling some obscure number to turn it off instead of there being a proper toggle in the ini was also a really bad choice.

I think you guys should just consider the user experience a bit more when you implement stuff like this lol. 

Like instead of completely removing this due to "pushback" you guys should actually go through the feedback people have been giving and take it into account.

**Comment by Ash-735** [comment]
> > Like instead of completely removing this due to "pushback" you guys should actually go through the feedback people have been giving and take it into account.

A lot of people have advocated that they want the snow gone and sent threats about it calling us idiots, etc, stating that they've downloaded a fix mod, not a new features mod, which is odd because things like CHSS shadows and sun shafts were not in the original game either, so if we were to listen to that advice does that mean we should remove those features too? They wern't in the original game after all.

**Comment by juicermv** [comment]
> A lot of people have advocated that they want the snow gone and sent threats about it calling us idiots, etc, stating that they've downloaded a fix mod, not a new features mod, which is odd because things like CHSS shadows and sun shafts were not in the original game either, so if we were to listen to that advice does that mean we should remove those features too? They wern't in the original game after all.

Obviously not. You're doing a bit of a strawman here. I'm saying you can take the constructive criticism about how you guys implemented the snow (which is basically just to make it optional) and listen to it. 

Obviously all the other mean-spirited and bad faith stuff can be tossed aside. It's not a real representation of how people feel. 

I mean FF has like what, thousands of users? 
And how many have sent toxic complaints, less than a dozen? Let's put things in perspective.

**Comment by dymck1** [comment]
I just want to chime in and say that despite my initial confusion, once I actually realized that it was a snow effect I really enjoyed it. The effect is very well done imo and blends in great with the games graphics, everything from the snow dirt on the cars to little piles of snow on windowsills, it looks great. And when I stepped out once from my apartment and snow was actually falling was really awesome, great atmosphere and added a lot to my experience. Driving through the falling snow with the jazz station on, it was really special. It's especially good for this time of year, obviously. I think what's happened, at least what happened to me, is that the steam winter sale is going on, so a bunch of people are buying the game for 5 bucks and playing it for the first time now, hence a lot of confusion, because stepping out for the first time in the game unawares and seeing the ground all white just immediately reads as a glitch on first impression. Although like I said once I realized what was going on I found the effect to be very well done.

So basically I would say to please not remove the effect wholesale for future updates, maybe just find a way to explain what's happening to the user, I do like the idea of turning it on for this time of year though, so I'm not sure how to do it. Any aggressive tone or hate towards people who are doing great work on a mod that greatly enhances the experience of playing the game FOR FREE just doesn't make sense to me. For real I played for like 30 mins without fusion fix when I first installed and it was an absolute glitchy disaster lol, this mod fixed everything.

**Comment by ThirteenAG** [comment]
The plan was to implement seasonal events toggle, after this event ends, and when the next update is ready.
